tests/staticfiles_tests/test_storage.py

```diff
@@ -446,3 +537,44 @@ def test_collect_static_files_subclass_of_static_storage(self):
         dir_mode = os.stat(test_dir)[0] & 0o777
         self.assertEqual(file_mode, 0o640)
         self.assertEqual(dir_mode, 0o740)
+
+
+@override_settings(
+    STATICFILES_STORAGE='django.contrib.staticfiles.storage.CachedStaticFilesStorage',
+)
+class TestCollectionHashedFilesCache(CollectionTestCase):
+    """
+    Test changing an image file and make sure that css files referring to that file
```

Comment:
State the expected behavior as per https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style. Also wrap docstrings at 79 chars.